### PR TITLE
KAA-1389: C Code Style update

### PR DESCRIPTION
**Pull request by rasendubi**

### CODE REVIEW

**Comment by vadimol** [comment]
Failed due to KAA-1104

**Comment by vadimol** [comment]
@kaa-leeroy retest this please

File: client/client-multi/client-cpp/kaa/log/ILogCollector.hpp [hunk 1]
```diff
@@ -90,8 +90,6 @@ class ILogCollector {
 
     /**
      * @brief Set a listener which receives a delivery status of each log bucket.
```
Comment (akokhanovskyi) [inline client/client-multi/client-cpp/kaa/log/ILogCollector.hpp:92]: What is this change doing in the C Code Style update?

**Comment by akokhanovskyi** [comment]
What's the ticket number for this documentation change? Please open a task if there's none. We should not accept any PRs without a ticket that would go into the release notes.

**Comment by rasendubi** [comment]
@akokhanovskyi, opened a ticket: [KAA-1389](http://jira.kaaproject.org/browse/KAA-1389)

File: client/client-multi/client-cpp/kaa/log/ILogCollector.hpp (hunk 1 above)
Comment (rasendubi) [reply client/client-multi/client-cpp/kaa/log/ILogCollector.hpp:92]: Mostly because that's what provoked changes to the style guide.

**Comment by forGGe** [comment]
+

**Pull request merged**